VideoLocker/src/main/java/org/edx/mobile/view/DiscussionAddCommentFragment.java

```diff
@@ -149,7 +149,6 @@ public void onException(Exception ex) {
                 buttonAddComment.setEnabled(true);
             }
         };
-        createCommentTask.setTaskProcessCallback(null);
```

Comment:
The progress callback at least should be set to `null` in these modules, since we're utilizing the button's own integrated progress spinner instead of the generic spinner overlay.